diagnostics/gpu.md

```diff
@@ -0,0 +1,15 @@
+# Graphics and GPU info
+GPU logs include details on the user's GPU and any potential graphics or rendering issues. If your WV2 has been created successfully but the content is not there, it might be due to a hosting and/or GPU driver issue.
+
+There's a few pieces of information that can help us investigate these issues:
+## DirectX Details
+On the device that is hitting the issue:
+1. Run `dxdiag` from a console window
+1. Once the dialog displays and is done capturing info (there is a small progress bar) hit the "Save All Information" button to save the info to a `dxdiag.txt` file
+1. Share the `dxdiag.txt` file
+
+## GPU Logs
+1. Navigate your WV2 to `edge://gpu`
```

Comment:
Might want to note that this navigation has be to built into the app code to call Navigate on WebView2 object and cannot be done via DevTools console by setting window.location.href.